src/main/java/org/opentripplanner/middleware/tripMonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -451,86 +567,91 @@ public boolean shouldSkipMonitoredTripCheck() throws Exception {
         return true;
     }
 
-    /**
-     * Calculate the next possible time that the itinerary is possible, advancing the targetZonedDateTime and setting
-     * the targetZonedDateTime, targetDate, otpResponse and matchingItinerary. Returns false if an error occurs or a
-     * matching itinerary isn't found on the next possible date the trip should be monitored on.
-     */
-    private boolean calculateNextItinerary() {
-        // double check that this trip is in fact active and throw an error if it is not
-        if (trip.isInactive()) {
-            throw new RuntimeException("An attempt was made to find the next possible active date of an inactive monitored trip.");
-        }
-
-        // if the trip is not active on the current zoned date time, advance until a day is found when the trip is
-        // active.
+    private void advanceToNextActiveTripDate() throws URISyntaxException {
```

Comment:
Woops, missed a spot that might benefit from some javadoc.

Reply:
Added in 1987213b6754ffa98f4748820b628be64f9443f4